omnibus/files/private-chef-cookbooks/private-chef/libraries/private_chef.rb

```diff
@@ -642,7 +642,7 @@ def consume_existing_secrets
       existing_secrets.each do |k, v|
         v.each do |pk, p|
           if not PrivateChef[k]
-            Chef::Log.info("Ignoring unused secret for #{k}.")
+            Chef::Log.warn("The secret for #{k} doesn't appear to be valid, you should remove it.")
```

Comment:
I think we want some combination of these two messages.  "doesn't appear to be valid" doesn't tell the user much about what is wrong.  I would say "is not used by the Chef Server, ..."